server/json-logger.cpp

```diff
@@ -185,8 +185,9 @@ JsonLogger::JsonBuffer &JsonLogger::JsonBuffer::append_raw_string(vk::string_vie
   return *this;
 }
 
-void JsonLogger::init(int64_t release_version) noexcept {
+void JsonLogger::init(const int64_t release_version, const vk::string_view hostname) noexcept {
```

Comment:
No need for `const` here, but up to you

Reply:
i believe it is good practice to use `const'